src/org/omegat/gui/glossary/DefaultGlossaryRenderer.java

```diff
@@ -145,7 +141,9 @@ protected void render(GlossaryEntry entry, IRenderTarget<?> trg) {
                 trg.append(", ");
             }
 
-            trg.append(bracketEntry(targets[i]), priorities[i] ? PRIORITY_ATTRIBUTES : null);
+            SimpleAttributeSet attrs = new SimpleAttributeSet (priorities[i] ? PRIORITY_ATTRIBUTES : NO_ATTRIBUTES);
```

Comment:
There is an unneeded space after `SimpleAttributeSet`.

Just FYI, we have a Gradle task to auto-format files changed on the current branch:

```
./gradle spotlessChangedApply
```

This is based on our format config available here: https://github.com/omegat-org/omegat/blob/master/docs_devel/eclipse-formatting.xml

Reply:
After your comments in the other pull request, I now use gradle task "check" and read the report in HTML about the code I modified. Sorry that I did not see this one.  This is now corrected.

Thanks for the tip about global task spotlessChangedApply. That said, if it automatically changes all classes, including those I did not modify, then I prefer to apply the change in a dedicated commit, rather than including the change in a functional commit. In my work I often use gitk to check when and where I implemented a dedicated feature (that's what I do when you ask me for a feature implemented in DGT) and to do that, I prefer to have atomic commits, instead of seeing in a big commit changes which are not related to the feature described in commit's comment. 
I accept to correct the style in my code before pushing in GitHub, so I should not add new style issues. But correcting already existing issues is another topic, which should be in own commit, whenever you do it automatically or not.

Reply:
> That said, if it automatically changes all classes, including those I did not modify

Unfortunately it can't be applied to classes, only files. It's only supposed to touch files you changed, but there are probably situations where it gets it wrong.

> But correcting already existing issues is another topic, which should be in own commit, whenever you do it automatically or not.

Yes. I want to simply apply it to the whole codebase, but I imagine that would make accepting patches from DGT very hard, so I have not done it yet. Maybe some time in the future.